### PR TITLE
Loadout Blindfolds

### PR DESCRIPTION
**Pull request by Snakebittenn**
Adds the old one, and adds one more friendly for recolors.

### CODE REVIEW

**Comment by Snakebittenn** [comment]
!review

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -835,3 +835,15 @@ BLIND     // can't see anything
 	else
 		flash_protection = FLASH_PROTECTION_NONE
 		tint = TINT_NONE
+
+/obj/item/clothing/glasses/whiteblindfold
```
Comment (NonQueueingMatt) [inline code/modules/clothing/glasses/glasses.dm:839]: Why is this not just a subtype of `/obj/item/clothing/glasses/sunglasses/blindfold`?

**Review by Chada-1 — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm [hunk 2]
```diff
@@ -104,3 +104,17 @@
...
+	description = "A selection of blindfolds."
+	path = /obj/item/clothing/glasses/sunglasses/blindfold
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/eyes/blindfolds/New()
+	..()
+	var/blindfold = list()
+	blindfold["blindfold, black"] = /obj/item/clothing/glasses/sunglasses/blindfold
+	blindfold["blindfold, white"] = /obj/item/clothing/glasses/sunglasses/blindfold/white
+	blindfold["white blindfold, see-through"] = /obj/item/clothing/glasses/sunglasses/blindfold/white/seethrough
```
Comment (MarinaGryphon) [inline code/modules/client/preference_setup/loadout/loadout_eyes.dm]:
```suggestion
	blindfold["blindfold, transparent white"] = /obj/item/clothing/glasses/sunglasses/blindfold/white/seethrough
```
For parallelism.

**Review by MarinaGryphon — CHANGES_REQUESTED** [review]

**Review by MarinaGryphon — APPROVED** [review]
My requested changes got sniped. Wtf.

File: code/modules/clothing/glasses/glasses.dm (hunk 1 above)
Comment (Snakebittenn) [reply code/modules/clothing/glasses/glasses.dm]: got it

**Pull request merged**